google/data_source_google_compute_default_service_account.go

```diff
@@ -0,0 +1,44 @@
+package google
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceGoogleComputeDefaultServiceAccount() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceGoogleComputeDefaultServiceAccountRead,
+		Schema: map[string]*schema.Schema{
+			"email": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"project_id": {
```

Comment:
For consistency with all the other resource, could you rename this field "project"